lib/p2p/packets/Packet.ts

```diff
@@ -40,6 +40,8 @@ abstract class Packet<T = any> implements PacketInterface {
   public abstract get direction(): PacketDirection;
   public body?: T;
   public header: PacketHeader;
+  /** Wire protocol delimiter. */
+  public static PROTOCOL_DELIMITER = 'ↂ₿ↂ';
```

Comment:
Nice! What are those 2 things on the side?

Reply:
Just some random thing I found on the unicode table. I think it's a roman number. Feel free to change the delimiter if you want.

Reply:
Nah it's fine, as long as it works.